### DISCUSSION TITLE
Error in Prediction - Training and Inference

### DISCUSSION THREAD

**Discussion by a8cone** [talmolab/sleap#1614] [Help!] [answered]
Hello,

I am getting an error before/during the Run Inference function. 

I attached photos of the errors I am receiving. 

Any suggestions on how I should continue or fix this would be helpful.
![image](https://github.com/talmolab/sleap/assets/60786631/cf087f7b-2e58-4243-9c46-ec0429e1ee0e)
![image](https://github.com/talmolab/sleap/assets/60786631/96a04151-f3e9-4ba9-afed-1129d53c0195)
![image](https://github.com/talmolab/sleap/assets/60786631/a3cdb7e0-bf59-441c-a9cc-1701308a1ebf)
![image](https://github.com/talmolab/sleap/assets/60786631/397a6c0c-7c45-49c2-9ca6-a9b17037171b)

Thank you.

AC

Software versions:
SLEAP: 1.1.5
TensorFlow: 2.3.1
Numpy: 1.18.5
Python: 3.6.13
OS: Windows-10-10.0.19041-SP0

**Comment by talmo** [comment, marked as answer]
Hi @a8cone,

I believe this might be a bug that we fixed a while back, do you mind trying to update SLEAP to [v1.3.3](https://github.com/talmolab/sleap/releases/tag/v1.3.3)?

Let us know if you're still seeing the issue after that!

Cheers,

Talmo
